.github/workflows/ros2.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       run: |
         # TODO(ros-tooling/setup-ros-docker#7): calling chown is necessary for now
         sudo chown -R rosbuild:rosbuild "$HOME" .
+    - name: test-git-secret-actions
+      uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.3.3
```

Comment:
We'll need to use a newer version of the `aws-robomaker-github-actions` GitHub repo.